src/Illuminate/Http/Concerns/InteractsWithInput.php

```diff
@@ -296,6 +296,17 @@ public function boolean($key = null, $default = false)
         return filter_var($this->input($key, $default), FILTER_VALIDATE_BOOLEAN);
     }
 
+    /**
+     * Retrieve input from the request as a collection.
+     *
+     * @param  string|null  $key
+     * @return \Illuminate\Support\Collection
+     */
+    public function collect($key = null)
+    {
+        return collect($this->input($key, []));
```

Comment:
I think the default value can be omitted. `$this->input(...)` will return `null` when the key does not exist and we get an empty collection either way.

Reply:
I believe you are correct, and I updated the PR